knowledge-base/kb-telerik-wpf-controls-build-platform.md

```diff
@@ -0,0 +1,35 @@
+---
+title: WPF Telerik controls are not displayed correctly in x64 build platform
+description: In some version of Visual Studio the designer does no display WPF Telerik controls correctly when the build platform is x64
```

Comment:
Keep is as simple as possible. Use keywords in the context of the user. Avoid fillings like "This article shows" or "In some version" in this case.

An alternative description could be: "Visual Studio design-time does not show Telerik UI elements when the platform is x64 bit". You will notice that the sentence is shorter and it replaces few of the words from the title in order to optimize the search engine. Those are "designer-->design-time", "display-->show", "controls-->elements". Additionally, I've added the word "bit".